.github/yaml_linter.json

```diff
@@ -1,18 +0,0 @@
-{
-    "problemMatcher": [
-        {
-            "owner": "yaml_linter",
-            "pattern": [
-                {
-                    "regexp": "^\\[HOOK\\s-\\syaml_linter\\.py\\]\\spre_export\\(\\):\\s(.+):(\\d+):(\\d+):\\s\\[(\\S+)\\]\\s(.+)\\s\\((.+)\\)$",
```

Comment:
I think we can keep this problem matcher in order to show directly in the webview of the files where the error is.
the regexexp just has to be shortened: `^(.+):(\\d+):(\\d+):\\s\\[(\\S+)\\]\\s(.+)\\s\\((.+)\\)$`

Reply:
ok, I did not know github had some default matchers